charters/DeiCharter.md

```diff
@@ -0,0 +1,31 @@
+# Diversity Equity and Inclusion Committee Charter
+
+Project Jupyter welcomes and encourages participation in our community by people of all backgrounds and identities. We are committed to promoting and sustaining a culture that values mutual respect, tolerance, and learning. We work together as a community to help each other live out these values.
+
+The Diversity Equity & Inclusion (DEI) Committee is a standing committee established under Jupyter’s governance model. The committee reviews its charter annually and may make changes at any time during the year. Substantive changes to the purpose or mission must be submitted to the Executive Council (EC) and the Software Steering Council (SSC) for approval.
+
+## Purpose 
+
+The purpose of this committee is to lead strategic initiatives designed to improve the diversity, equity, and inclusion of Project Jupyter community leaders, core developers, and end users across the globe. For the long-term sustainability of our community, we champion efforts to build an open source environment where people feel respected, valued, and encouraged to continue participating. 
```

Comment:
```suggestion
The purpose of this committee is to lead strategic initiatives designed to improve the diversity, equity, and inclusion of Project Jupyter community leaders, contributors, and end users across the globe. For the long-term sustainability of our community, we champion efforts to build an open source environment where people feel respected, valued, and encouraged to continue participating. The DEI group impacts all Jupyter forums, including but not limited to software, working groups, standing committees, JupyterCon, social media, GitHub, the EC, and the SSC.
```

(Including @afshin 's suggestion and addressing @ellisonbg and @fperez 's request)